runner/check-aws-ecs-tasks.js

```diff
@@ -26,13 +27,52 @@ async function main() {
       console.log('HALT')
       console.log('More than one task already running')
       console.log(data.taskArns)
+      const retryDelayMs = 300000 // 300k ms = 5 mins
+      sendNotification(data.taskArns, retryDelayMs)
     } else {
       console.log('OK')
       console.log('Only one running task found (assumed to be self)')
     }
   }
 }
 
+function sendNotification(taskArns, retryDelayMs = -1) {
+  const url = process.env.DISCORD_WEBHOOK_URL
+  const options = {
+    method: 'POST',
+    headers: {
+      "Content-Type": "application/json"
+    }
+  }
+  const message = [
+    {
+      title: 'CAS still running',
+      description: `A new CAS anchor task was not started because there is already at least one running.`,
+      color: 16776960,
+      fields: [
+        {
+          name: 'ARNs for Running CAS Anchor Tasks',
+          value: `${JSON.stringify(data.taskArns)}`,
```

Comment:
Could this be easily made into a link to the task logs rather than just the name of the task?

Reply:
Added that